logstash-core-event-java/src/main/java/org/logstash/Event.java

```diff
@@ -338,22 +348,33 @@ private Timestamp initTimestamp(Object o) {
             logger.warn("Error parsing " + TIMESTAMP + " string value=" + o.toString());
         }
 
-        tag(TIMESTAMP_FAILURE_TAG);
-        this.data.put(TIMESTAMP_FAILURE_FIELD, o);
-
-        return Timestamp.now();
+        return null;
     }
 
     public void tag(String tag) {
-        List<Object> tags = (List<Object>) this.data.get("tags");
-        if (tags == null) {
+        List<Object> tags;
+        Object _tags = this.getField("tags");
```

Comment:
2 things:
- There is not really any `_` prefix convention in Java but I sometimes use var names starting with underscore to note a temporary holder for the same var name without the underscore or to note a "throwaway" tmp var, all in very local contexts.
- I personally like it and find it concise. This is really a very **minor** style issue. Given the insignificant nature of this it is mostly a question of preference and since I wrote the code I think I can opt on keeping it.